transformer/transformer.go

```diff
@@ -223,7 +223,11 @@ func InitTransformers(transformerToInit map[string]string, selector labels.Selec
 		}
 	}
 	for _, selectedTransformerName := range selectedTransformerNames {
-		transformerConfig := transformerConfigs[selectedTransformerName]
+		transformerConfig, ok := transformerConfigs[selectedTransformerName]
+		if !ok {
+			logrus.Errorf("failed to find the config for the transformer with the name: '%s'", selectedTransformerName)
```

Comment:
```suggestion
			logrus.Errorf("failed to find the transformer with the name: '%s'", selectedTransformerName)
```

config of a transformer might mean config section in the transformer yaml.